running_wtr_mbe_d_scf.py

```diff
@@ -0,0 +1,94 @@
+import time
+import pickle
+from mbe_density import *
+
+wtr_mbe_d_scf = {}
+filename = "wtr_mbe_d_scf.py"
+bs = "aug-cc-pvdz"
+
+root_dir  = os.getcwd()
+geom_dir  = os.path.join(root_dir, 'geometries')
+n = list(range(30))
+
+if "Trimer" not in wtr_mbe_d_scf:
+    file_path = os.path.join(geom_dir, "Trimer.xyz")
+    mol = load_molecule(file_path)
+    mono2atom = [n[0:3], n[3:6], n[6:9]]
+    monomers = make_nmers(mono2atom, mol, 1)
+    dimers = make_nmers(mono2atom, mol, 2)
+    guess = make_guess(compute_densities(monomers, bs), compute_densities(dimers, bs), mol, mono2atom, bs)
+    wtr_mbe_d_scf["Trimer-mbe_d_g"] = run_scf(mol, bs, guess)
+    wtr_mbe_d_scf["Trimer-atom_g"] = run_scf(mol, bs)
+    
+if "Tetramer" not in wtr_mbe_d_scf:
+    file_path = os.path.join(geom_dir, "Tetramer.xyz")
+    mol = load_molecule(file_path)
+    mono2atom = [n[0:3], n[3:6], n[6:9], n[9:12]]
+    monomers = make_nmers(mono2atom, mol, 1)
+    dimers = make_nmers(mono2atom, mol, 2)
+    guess = make_guess(compute_densities(monomers, bs), compute_densities(dimers, bs), mol, mono2atom, bs)
+    wtr_mbe_d_scf["Tetramer-mbe_d_g"] = run_scf(mol, bs, guess)
+    wtr_mbe_d_scf["Tetramer-atom_g"] = run_scf(mol, bs)
+    
+if "Pentamer" not in wtr_mbe_d_scf:
+    file_path = os.path.join(geom_dir, "Pentamer.xyz")
+    mol = load_molecule(file_path)
+    mono2atom = [n[0:3], n[3:6], n[6:9], n[9:12], n[12:15]]
+    monomers = make_nmers(mono2atom, mol, 1)
+    dimers = make_nmers(mono2atom, mol, 2)
+    guess = make_guess(compute_densities(monomers, bs), compute_densities(dimers, bs), mol, mono2atom, bs)
+    wtr_mbe_d_scf["Pentamer-mbe_d_g"] = run_scf(mol, bs, guess)
+    wtr_mbe_d_scf["Pentamer-atom_g"] = run_scf(mol, bs)
+    
+#if "Hexamer_Prism" not in wtr_mbe_d_scf:
```

Comment:
My computer still freezes with this particular geometry.

Reply:
but you can run water 7 through 10?

Reply:
Here's something else: the pentamer geom isn't setup as a sequence of waters. It has all of the oxygens listed first, then all of the hydrogens. So `mono2atom` for it is not correct. Either the xyz file needs to be reordered, or `mono2atom` needs to be changed accordingly.

Reply:
Also, the geoms for the clusters larger than hex aren't included in the current state of the PR.

Reply:
The reordered geometries were in the last PR, including the pentamer.

Reply:
Yes, heptamer through decamer can be run Ryan.

Reply:
Jonathan I'm using the reordered pentamer that was on the last PR.

Reply:
If it's just the hexamer data point that won't run then I'd recommend skipping it. It's super weird that it won't run though.

Reply:
Yeah. I just don't have an answer for you on that.

Reply:
I would venture a guess and say that it would take 7 cycles to converge, and that the scf time would be near evenly between pentamer and heptamer given the density timings for monomers and dimers.